lib/common/monitoring/alarms/UsageAlarmFactory.ts

```diff
@@ -75,6 +82,31 @@ export class UsageAlarmFactory {
     });
   }
 
+  /**
+   * Allows addition of multiple metrics from the same class.
+   * @param percentMetric 
+   * @param usageType Will be added to Alarm name
+   * @param props 
+   * @param disambiguator 
+   * @returns 
+   */
+  addMaxCpuUsagePercentAlarms(
```

Comment:
This naming doesn't really make sense, it's still only adding a single alarm.

What do you think about adding an additional argument for `usageType` to the existing function and retaining existing behaviour if not specified?